python_modules/dagster/dagster_tests/core_tests/resource_tests/test_resource_parameters.py

```diff
@@ -303,3 +304,40 @@ def the_asset(context, config: AnAssetConfig, foo: ResourceOutput[str]):
         .success
     )
     assert executed["the_asset"]
+
+
+@pytest.mark.skipif(sys.version_info < (3, 9), reason="requires python3.9")
+def test_no_err_builtin_annotations():
+    # Ensures that we can use Python builtin types without causing any issues, see
+    # https://github.com/dagster-io/dagster/issues/11541
+
+    # Use aliases for the builtin types so that this doesn't cause a load error on
+    # python < 3.9
+    _list = list
+    _dict = dict
```

Comment:
Why is this alias necessary for the this test to pass? I'm not understanding.

Reply:
Ah yeah, I think it's actually not necessary, since the guard will prevent the function body & type annotations from being evaluated

Reply:
Great thanks